.github/workflows/kserve_ci_cpu.yml

```diff
@@ -0,0 +1,40 @@
+name: KServe CI CPU
+
+on:
+  workflow_dispatch:
+  # runs everyday  at 5:15am
+  schedule:
+    - cron:  '15 5 * * *'
```

Comment:
instead of running this nightly you can run it only when changes to kubernetes folder are made

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Reply:
Since we push nightly images, we should be running this nightly. It takes < 5 minutes to run